ci-operator/config/openshift/cluster-nfd-operator/openshift-cluster-nfd-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  base:
+    name: "4.2"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.10
```

Comment:
Shouldn't this be something newer?

Reply:
The golang version? I just followed what other operators in the config repository are using.